File: samples/setup/Setup.WPF.Core/ViewModels/ShellViewModel.cs
using Caliburn.Micro;

namespace Setup.WPF.Core.ViewModels
{
    public class ShellViewModel : Conductor<object>
    {
        private string _title;

        public string Title
        {
            get => _title;
            set
            {
                _title = value;
                NotifyOfPropertyChange("Title");
            }
        }

        public ShellViewModel()
        {
            Title = "Welcome to Caliburn Micro in .net 8 WPF";
        }

    }
}

File: samples/setup/Setup.WPF.Core.VB/ViewModels/ShellViewModel.vb
Imports Caliburn.Micro

Public Class ShellViewModel
    Inherits Screen

    Private _title1 As String

    Public Property Title As String
        Get
            Return _title1
        End Get
        Set
            _title1 = Value
        End Set
    End Property

    Public Sub New()
        Title = "Welcome to Caliburn Micro in WPF using VB and .net 5"
    End Sub
End Class
